Pipfile

```diff
@@ -11,7 +11,7 @@ requests = "*"
 
 [dev-packages]
 "flake8" = "*"
-"pep8" = "*"
+"pycodestyle" = "*"
```

Comment:
would you update the pipfile.lock as well? `pipenv install -d` should do that for you.

Reply:
btw, is this transparente? flake8 will take the new pycodestyle instead of pep8? Or do we need to do something else?

I can't test it right now, but will do later today.

Reply:
just tested, it's not transparent:

```
flake8.exceptions.FailedToLoadPlugin: Flake8 failed to load plugin "flake8_regex" due to No module named 'pep8'.
```

AFAIU this is all about this warning. Note it's not an error, it doesn't block the linting or git hooks:

```
Linting Python files with Flake8
/Users/gustavo/Development/git/sgqlc/.venv/lib/python3.7/site-packages/pep8.py:110: FutureWarning: Possible nested set at position 1
  EXTRANEOUS_WHITESPACE_REGEX = re.compile(r'[[({] | []}),;:]')
FLAKE8 SUCCEEDED
```

Reply:
`flake8-regex` appears to be unmaintained.  https://github.com/aristide-n/flake8-regex

Reply:
I'll see if I can tackle this one later this week. If you can give it a try, let me know.

The original PR couldn't be merged since it required on manual tweaks to.

Reply:
flake8-regex wasn't used and seems unmaintained, so we removed it and with that could also remove the pep8 (which was deprecated in favor of pycodestyle).